src/main.ts

```
@@ -5,6 +5,7 @@ import * as dotenv from 'dotenv';
 import { ValidationPipe } from '@nestjs/common';
 import { HttpExceptionFilter } from './filters/http-exception.filter';
 import { LoggerService } from './modules/logging/services/logger.service';
+import { createClient } from 'redis';
```

ESLint

[error] 'createClient' is defined but never used.

(@typescript-eslint/no-unused-vars)